GooglePlayInstant/LoadingScreenEngine/LoadingScreenConfig.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
```

Comment:
File is missing license info and class/field documentation.

Reply:
Resolved.